volume/volume.go

```diff
@@ -136,6 +136,8 @@ type QuiesceDriver interface {
 type CloudBackupDriver interface {
 	// CloudBackupCreate uploads snapshot of a volume to the cloud
 	CloudBackupCreate(input *api.CloudBackupCreateRequest) error
+	// CloudBackupGroupCreate creates and then uploads volumegroup snapshots
+	CloudBackupGroupCreate(input *api.CloudBackupGroupCreateRequest) error
```

Comment:
How is status determined? When the user calls CloudBackupCreate( src_volume_id ) they can then call CloudBackupStatus( src_volume_id )

Is there something similar for this new API?